features/schema_metadata.feature

```diff
@@ -0,0 +1,303 @@
+    Scenario: Data types can be determined by the column metadata for Cassandra
+              versions 2.0+
+    Given the following example:
+      """php
+      <?php
+      $cluster   = Cassandra::cluster()
+                         ->withContactPoints('127.0.0.1')
+                         ->build();
+      $session   = $cluster->connect("simplex");
+      $schema    = $session->schema();
+      $id        = $schema->keyspace("simplex")->table("values")->column("id");
+      $bigint    = $schema->keyspace("simplex")->table("values")->column("bigint_value");
+      $decimal   = $schema->keyspace("simplex")->table("values")->column("decimal_value");
+      $double    = $schema->keyspace("simplex")->table("values")->column("double_value");
+      $float     = $schema->keyspace("simplex")->table("values")->column("float_value");
+      $int       = $schema->keyspace("simplex")->table("values")->column("int_value");
+      $varint    = $schema->keyspace("simplex")->table("values")->column("varint_value");
+      $ascii     = $schema->keyspace("simplex")->table("values")->column("ascii_value");
+      $text      = $schema->keyspace("simplex")->table("values")->column("text_value");
+      $varchar   = $schema->keyspace("simplex")->table("values")->column("varchar_value");
+      $timestamp = $schema->keyspace("simplex")->table("values")->column("timestamp_value");
+      $blob      = $schema->keyspace("simplex")->table("values")->column("blob_value");
+      $uuid      = $schema->keyspace("simplex")->table("values")->column("uuid_value");
+      $timeuuid  = $schema->keyspace("simplex")->table("values")->column("timeuuid_value");
+      $inet      = $schema->keyspace("simplex")->table("values")->column("inet_value");
+      $list      = $schema->keyspace("simplex")->table("values")->column("list_value");
+      $map       = $schema->keyspace("simplex")->table("values")->column("map_value");
+      $set       = $schema->keyspace("simplex")->table("values")->column("set_value");
+
+      echo $id->type()->name() . "\n";
+      echo $bigint->type()->name() . "\n";
+      echo $decimal->type()->name() . "\n";
+      echo $double->type()->name() . "\n";
+      echo $float->type()->name() . "\n";
+      echo $int->type()->name() . "\n";
+      echo $varint->type()->name() . "\n";
+      echo $ascii->type()->name() . "\n";
+      echo $text->type()->name() . "\n";
+      echo $varchar->type()->name() . "\n";
+      echo $timestamp->type()->name() . "\n";
+      echo $blob->type()->name() . "\n";
+      echo $uuid->type()->name() . "\n";
+      echo $timeuuid->type()->name() . "\n";
+      echo $inet->type()->name() . "\n";
+      echo $list->type()->name() . "\n";
+      echo $list->type()->type()->name() . "\n";
+      echo $map->type()->name() . "\n";
+      echo $map->type()->keyType()->name() . "\n";
+      echo $map->type()->valueType()->name() . "\n";
+      echo $set->type()->name() . "\n";
+      echo $set->type()->type()->name() . "\n";
```

Comment:
Yeah, I was going back and forth with that at well. Since the `__toString()` is calling those functions anyway I don't see a reason why not.